scripts/configs/training/training_params.yaml

```diff
@@ -33,7 +33,8 @@ unfreezing_epoch: 25
 dropout_rate: 0.1
 
 # min training epochs; training cannot terminate before. compare to unfreezing_epoch, check_val_every_n_epoch, early_stop_patience
-min_epochs: 100
+min_epochs: 300 
```

Comment:
was this important? We did not find it especially important

Reply:
You are probably correct! I just wanted it to train longer cause I thought it wasn't fully converged. Maybe I was wrong about this.